hw01.md

```diff
@@ -0,0 +1,16 @@
+# Lindsay Cooper 
+Hmwk 1
+
+I am a MAPSS student concentrating in psychology. I am especially interested in clinical and community psychology. I recently moved to Chicago from Boston. I am originally from Seattle. I was a preschool teacher for several years prior to joining MAPSS.
+
+My favorite spot in Chicago (so far):
+<em> https://www.artic.edu <em>
+
+**Hobbies Include:**
```

Comment:
From what I'm seeing in the presentation format of the code, it seems like you were able to get both bold and italics.. I'm wondering how?

Reply:
Hi!

Thanks so much for your comments! 

I am not sure how/why that text remained bold and italics w/o accompanying code.  But, I think the code is something like this:

*__Hmwk 1__*

(combing the asterisks with the dashes) 

*italics* and _italics_
**bold** and __bold__ 

Source:
https://rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf

Reply:
sorry! I just realized that it didn't print my code as code. 

But, you can use the asterisks for bold and the underscore for italics, or vice versa.